tests/e2e.test.ts

```diff
@@ -112,6 +114,39 @@ describe('E2E tests', function () {
     }
   });
 
+  it('has the correct secret vars', function () {
+    if (SECRET_ENV && service) {
+      const expected = parseEnvVars(SECRET_ENV);
+      const containers = _.get(service, 'spec.template.spec.containers');
+      const actual = containers[0]?.env;
+      expect(actual).to.have.lengthOf(expected.length);
+      actual.forEach((secretEnvVar: run_v1.Schema$EnvVar) => {
+        const found = expected.find((expectedSecretEnvVar) =>
+          _.isEqual(secretEnvVar.name, expectedSecretEnvVar.name),
+        );
+        expect(found).to.not.equal(undefined);
+      });
+    }
+  });
+
+  it('has the correct secret volumes', function () {
+    if (SECRET_VOLUMES && service) {
+      const expected = parseEnvVars(SECRET_VOLUMES);
+      const containers: run_v1.Schema$Container[] = _.get(
+        service,
+        'spec.template.spec.containers',
+      );
+      const actual = containers[0]?.volumeMounts;
+      expect(actual).to.have.lengthOf(expected.length);
+      actual?.forEach((secretVolume: run_v1.Schema$VolumeMount) => {
+        const found = expected.find((expectedSecretVolume) =>
+          _.isEqual(secretVolume.mountPath, expectedSecretVolume.name),
+        );
+        expect(found).to.not.equal(undefined);
+      });
+    }
+  });
```

Comment:
Not entirely sure if this will work to assert mounted volumes, or if there could be more mounted volumes than the mounted secret. Either way, I guess the tests will show 🤷

Reply:
Error:
```
has the correct secret volumes:
     AssertionError: expected undefined to not equal undefined
      at /home/runner/work/deploy-cloudrun/deploy-cloudrun/tests/e2e.test.ts:145:30
```

it looks like the YAML for Volume Mounts truncates the mountPath:
```
    volumeMounts:
        - name: secret_value-mif-tet-xel
          readOnly: true
          mountPath: /api/secrets
```
And is supplemented by volumes:
```
volumes:
      - name: secret_value-mif-tet-xel
        secret:
          secretName: secret_value
          items:
          - key: latest
            path: my-secret
```

Reply:
Thanks for the debug data. I've added a lookup and matching to the volumes now as well, so hopefully this time it will fly 😃